docs/docs/references/CLI/rill/docs/docs.md

```diff
@@ -0,0 +1,16 @@
+## rill docs
+
+Manage documentation
+
+### Options inherited from parent commands
+
+```
+  -h, --help   Print usage
+```
+
+### SEE ALSO
+
+* [rill](../rill.md)	 - Rill CLI
+* [rill docs generate](generate.md)	 - Generate CLI documentation
+* [rill docs open](open.md)	 - Open docs.rilldata.com
```

Comment:
@begelundmuller : should we hide docs generate and open from referencing here ?

Reply:
Yeah – and also the `rill runtime` and `rill admin` commands (used internally to start those services). In general, all the commands where `Hidden: !cfg.IsDev()` is set should not be generated in the docs.